src/Aspire.Dashboard/wwwroot/css/app.css

```diff
@@ -49,7 +49,7 @@ fluent-toolbar[orientation=horizontal] {
 
     --error-ui-foreground-color: #000000;
     --error-ui-accent-foreground-color: #512bd4;
-    --reconnection-ui-bg: #D1D1D1;
+    --reconnection-ui-bg: #D6D6D6;
```

Comment:
I feel like this is probably still too bright. Did you update the screenshots in your PR description with the end state here? I'd expect the bg color to be largely unchanged, just an overlay that darkens what's there and makes is easier for the error content to float over the top.

Reply:
@drewnoakes we are not able to make it any darker with the current foreground colors. This is the exact limit - note that the actual visible color is darker due to the opacity

Reply:
Right, but the foreground colours are for light mode.

The thing I'm trying to avoid is flashing a large bright area in users' eyes, when they're in dark mode. A dark background with light foreground should achieve that, and meet contrast ratio requirements.